src/helper.jsx

```diff
@@ -0,0 +1,7 @@
+export const getCRSFToken = () => {
```

Comment:
file extension jsx not needed (not even a component), I think we should probably get rid of ```jsx``` extensions everywhere anyway I did the same in sdk recently